bin/node/src/commands.rs

```diff
@@ -84,6 +91,25 @@ fn open_keystore(
     }
 }
 
+fn bootstrap_backup(chain_params: &ChainParams, account_id: &AccountId) {
```

Comment:
TBH if we go with this approach so should also be `base_path`, `node_key_file`. Maybe we should introduce new argument `NodeParams` which contains those three things? Those things are all related to node and not to chain, so I would be against extracting only one of those.

Aaaand there is also stuff like `account_id` which maybe should be more of account params together with `account_seed` and `keystore_params`? Not sure about that.

Right now I think we should extract base_path, node_key_file, backup_dir to NodeParams, and leave the rest as it is.

Reply:
yup, sounds better than the current organization, you can give it a go

Reply:
On second thought this might need some more work, to systemize our whole CLI. I think that this PR consist enough critical changes and CLI modifications are one of them. I would prefer to add `backup_dir` to `ChainParams`/make it a separate param for now and create a special Jira story for cleaning up the whole CLI (there was for example one more problem regarding the format of how we save p2p_secret compared to Substrate). I can create and refine this task.

Reply:
okay, go with Jira then; I think you can refer to this example in the ticket description